src/openrct2/localisation/Formatting.cpp

```diff
@@ -588,7 +588,15 @@ namespace OpenRCT2
                 {
                     auto month = DateGetMonth(arg);
                     auto year = DateGetYear(arg) + 1;
-                    FormatMonthYear(ss, month, year);
+                    FormatMonthYear(ss, month, year, false);
+                }
+                break;
+            case FormatToken::MonthYearSentence:
+                if constexpr (std::is_integral<T>())
```

Comment:
(Note to self:) This check shouldn’t be necessary. Also fold in with previous condition.